src/chainparams.cpp

```diff
@@ -194,7 +194,8 @@ class CMainParams : public CChainParams {
         consensus.V19Height = 1899072; // 0000000000000015e32e73052d663626327004c81c5c22cb8b42c361015c0eae
         consensus.V20Height = 1987776; // 000000000000001bf41cff06b76780050682ca29e61a91c391893d4745579777
         consensus.MN_RRHeight = 2128896; // 0000000000000009a9696da93d3807eb14eb00a4ff449206d689156a21b27f26
-        consensus.MinBIP9WarningHeight = 2128896 + 2016; // mn_rr activation height + miner confirmation window
+        consensus.WithdrawalsHeight = 2201472; // 00000000000000210518749e17c00b035a2a4982c906236c28c41ea2231bf7ef
```

Comment:
@coderabbitai, can you validate that this is indeed the block height / hash where the DEPLOYMENT_WITHDRAWALS fork activated?

Reply:
Well; that's correct. this PR does indeed change the code to assume that the activation height / hash is as shown. However, can you validate this is the correct height / hash combo to set? IE was this the actual block on the dash network that activated the hard fork